addons/mass_mailing/static/src/js/mailing_contact_kanban.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import KanbanController from 'web.KanbanController';
+import KanbanView from 'web.KanbanView';
+import viewRegistry from 'web.view_registry';
+import {_t} from 'web.core';
+
+var MailingContactController = KanbanController.extend({
+    buttons_template: 'MailingContactKanbanView.buttons',
+
+    events: _.extend({}, KanbanController.prototype.events, {
+        'click .o_mass_mailing_import_contact': '_onImport',
+    }),
+
+    _onImport() {
+        this.do_action({
+            type: 'ir.actions.act_window',
+            name: _t('Import Mailing Contacts'),
+            views: [[false, 'form']],
+            res_model: 'mailing.contact.import',
+            target: 'new',
+            context: this.renderer.state.context,
+        });
```

Comment:
I would use the action you created instead :
```suggestion
        this.do_action('mass_mailing.mailing_contact_import_action'{
            additional_context: this.renderer.state.context,
        });
```